lib/OpenQA/Worker.pm

```diff
@@ -350,9 +351,35 @@ sub init ($self) {
             });
     }
 
+    if (   $global_settings->{IPMI_HOSTNAME}
+        && $global_settings->{IPMI_USER}
+        && $global_settings->{IPMI_PASSWORD}
+        && $global_settings->{IPMI_AUTOSHUTDOWN} ne '0')
+    {
+        log_info 'IPMI config present -> will periodically make sure SUT is powered off when unused.';
```

Comment:
Just a personal opipion. that should be better a debug log

Reply:
It's only printed once so better let the user know.
Also it's too important for debug.